imap_data_access/io.py

```diff
@@ -64,14 +64,12 @@ def download(file_path: Union[Path, str]) -> Path:
         # Construct the directory structure from the filename
         # This is for science files that contain the directory structure in the filename
         # Otherwise, we assume the full path to the file was given
-        parts = file_path.split("_")
-        instrument = parts[1]
-        datalevel = parts[2]
-        startdate = parts[4]
-        year = startdate[:4]
-        month = startdate[4:6]
-        destination = destination / instrument / datalevel / year / month
-    destination /= file_path
+
+        construct = imap_data_access.construct_upload_path(file_path)
+
+        destination = destination / construct
+    else:
+        destination /= file_path
```

Comment:
There was discussion about removing this special path for science files as well and forcing users to use the full path structure. Thoughts on removing this path entirely?